resources/buildConfigDefinitions.js

```diff
@@ -0,0 +1,162 @@
+  const firstComment = getCommentValue(last(property.leadingComments || []));
+  const name = property.key.name;
+  const prefix = getENVPrefix(iface);
+
+  if (!firstComment) {
+    return;
+  }
+  const components = firstComment.split(':ENV:').map((elt) => {
+    return elt.trim();
+  });
+  const help = components[0];
+  const env = components[1] || (prefix + toENV(name));
+  let type = property.value.type;
+  let isRequired = true;
+  if (type == 'NullableTypeAnnotation') {
+    isRequired = false;
+    type = property.value.typeAnnotation.type;
+  }
+  return {
+    name,
+    env,
+    help,
+    type,
+    types: property.value.types,
+    typeAnnotation: property.value.typeAnnotation,
+    required: isRequired
+  };
+}
+
+
+function doInterface(iface) {
+  return iface.body.properties.map((prop) => {
+    return processProperty(prop, iface);
+  }).filter((e) => e !== undefined);
+}
+
+function mapperFor(elt, t) {
+  const p = t.identifier('parsers');
+  const wrap = function(identifier) {
+    return t.memberExpression(p, identifier);
+  }
+
+  if (t.isNumberTypeAnnotation(elt)) {
+    return t.callExpression(wrap(t.identifier('numberParser')), [t.stringLiteral(elt.name)]);
+  } else if (t.isArrayTypeAnnotation(elt)) {
+    return wrap(t.identifier('arrayParser'));
+  } else if (t.isAnyTypeAnnotation(elt)) {
+    return wrap(t.identifier('objectParser'));
+  } else if (t.isBooleanTypeAnnotation(elt)) {
+    return wrap(t.identifier('booleanParser'));
+  } else if (t.isGenericTypeAnnotation(elt)) {
+    const type = elt.typeAnnotation.id.name;
+    if (type == 'Adapter') {
+      return wrap(t.identifier('moduleOrObjectParser'));
+    }
+    if (type == 'NumberOrBoolean') {
+      return wrap(t.identifier('numberOrBooleanParser'));
+    }
+    return wrap(t.identifier('objectParser'));
+  }
+}
+
+function inject(t, list) {
+  return list.map((elt) => {
+    if (!elt.name) {
+      return;
+    }
+    const props = ['env', 'help'].map((key) => {
+      if (elt[key]) {
+        return t.objectProperty(t.stringLiteral(key), t.stringLiteral(elt[key]));
+      }
+    }).filter((e) => e !== undefined);
+    if (elt.required) {
+      props.push(t.objectProperty(t.stringLiteral('required'), t.booleanLiteral(true)))
+    }
+    const action = mapperFor(elt, t);
+    if (action) {
+      props.push(t.objectProperty(t.stringLiteral('action'), action))
+    }
+    const obj = t.objectExpression(props);
+    return t.objectProperty(t.stringLiteral(elt.name), obj);
+  }).filter((elt) => {
+    return elt != undefined;
+  });
+}
+
+
+const plugin = function (babel) {
+  const t = babel.types;
+  const moduleExports = t.memberExpression(t.identifier('module'), t.identifier('exports'));
+  return {
+    visitor: {
+      Program: function(path) {
+        // Inject the parsers loader
+        path.pushContainer("body", t.importDeclaration([t.importNamespaceSpecifier(t.identifier('parsers'))],
+          t.stringLiteral('./parsers')));
```

Comment:
single quote on "body" -- "A foolish consistency is the hobgoblin of little minds" :).